cloudmanager/unauth_users.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 MongoDB Inc
```

Comment:
[np] Consistency with other files where you used 2019-Present?

Reply:
copy/paste probably 😅